pyodide_build/cli/venv.py

```diff
@@ -6,12 +6,114 @@
 from pyodide_build.out_of_tree import venv
 
 
+# TODO: disabled options that can be later supported have been commented out, fix them
+# --copies/--always-copy and symlink_app_data
 def main(
     dest: Path = typer.Argument(
         ...,
         help="directory to create virtualenv at",
     ),
+    clear: bool = typer.Option(
+        False,
+        "--clear/--no-clear",
+        help="Remove the destination directory if it exists",
+    ),
+    no_vcs_ignore: bool = typer.Option(
+        False,
+        "--no-vcs-ignore",
+        help="Don't create VCS ignore directive in the destination directory",
+    ),
+    symlinks: bool = typer.Option(
+        True,
+        "--symlinks",
+        help="Try to use symlinks rather than copies, when symlinks are not the default for the platform",
+    ),
+    # copies: bool = typer.Option(
+    #     False, "--copies", "--always-copy", help="Use copies rather than symlinks, even when symlinks are the default"
+    # ),
+    no_download: bool = typer.Option(
+        False,
+        "--no-download",
+        "--never-download",
+        help="Disable download of the latest pip/setuptools/wheel from PyPI",
+    ),
+    download: bool = typer.Option(
+        False,
+        "--download/--no-download",
+        help="Enable download of the latest pip/setuptools/wheel from PyPI",
+    ),
+    extra_search_dir: list[str] = typer.Option(
+        None,
+        "--extra-search-dir",
+        help="A path containing wheels to extend the internal wheel list",
+    ),
+    pip: str = typer.Option(
+        "bundle",
+        "--pip",
+        help="Version of pip to install as seed: embed, bundle, or exact version.",
+    ),
+    setuptools: str | None = typer.Option(
+        None,
+        "--setuptools",
+        help="Version of setuptools to install as seed: embed, bundle, none or exact version",
+    ),
+    wheel: str | None = typer.Option(
+        None,
+        "--wheel",
+        help="Version of wheel to install as seed: embed, bundle, none or exact version",
+    ),
```

Comment:
Let's leave off the `--wheel` option until we decide whether it would run in the native Python or the Pyodide Python.

Reply:
Thanks, dropped it in 3a9a17eb1230f67ac4a14c2a11d5300f62a0ba6f. This commit adds back the `--no-wheel` option, so the previous behaviour has been restored.